ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -150,7 +150,8 @@ const isRenewalCompleteForModule = (status: AccessModuleStatus) => {
   const wasBypassed = !!status?.bypassEpochMillis;
   return (
     wasBypassed ||
-    (isComplete && !isExpiringOrExpired(status?.expirationEpochMillis))
+    (isComplete &&
+      !isExpiringOrExpired(status?.expirationEpochMillis, status.moduleName))
```

Comment:
Does the new param requires optional chaining: `status?.moduleName`?

Reply:
`moduleName` has recently become a required element, so it must exist if `status` does